src/test/scala/beam/agentsim/agents/household/FastHouseholdCAVSchedulingSpec.scala

```diff
@@ -64,7 +64,6 @@ class FastHouseholdCAVSchedulingSpec
 
   describe("A Household CAV Scheduler") {
     it("generates two schedules") {
-      Skims.setup(services)
```

Comment:
Where is the injection set up in the tests?

Reply:
Now it is done in a regular Guice way during module definition in BeamHelper. https://github.com/LBNL-UCB-STI/beam/blob/4a4014f7e2df10372c45e03dfa7605ba7ad128aa/src/main/scala/beam/sim/BeamHelper.scala#L218